site/docs/transform/window.md

```diff
@@ -69,52 +69,171 @@ To replace any of the examples from total to `mean`, the operation just needs to
+                "as": "PercentOfTotal"
+            }
+        ],
+        "mark": { "type": "bar", "clip": true },
+        "encoding": {
+            "x": {
+                "field": "Activity",
+                "type": "nominal",
+                "scale": { "rangeStep": 12 },
+                "axis": { "title": "" }
+            },
+            "y": {
+                "field": "PercentOfTotal",
+                "type": "quantitative",
+                "axis": { "title": "% of total Time", "grid": false }
+            }
+        }
     }],
-    "frame": [null, null]
-  },
-  {
-    "calculate": "datum.x/datum.total",
-    "as": "Percent of Total"
-  }
+    "config": {
+        "view": { "stroke": "transparent" },
+        "axis": { "domainWidth": 1 }
+    }
+}
 ```
 
 
-### Difference from Total
+### Difference from Mean
+
+<div class="vl-example" data-name="window_transform_movie_mean_difference"></div>
 
 ```json
-  {
-    "window": [{
-        "op": "sum",
-        "field": "x",
-        "as": "total"
-    }],
-    "frame": [null, null]
-  },
-  {
-    "calculate": "datum.total - datum.x",
-    "as": "Difference from total"
-  }
+{
+    "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+    "description": "Bar graph showing how each movie is different from the average movie rating",
+    "width": 300,
+    "height": 50,
+    "data": { "url": "data/movies.json" },
+    "layer": [{
+        "transform": [{
+                "window": [{
+                    "op": "mean",
+                    "field": "IMDB_Rating",
+                    "as": "AverageRating"
+                }],
+                "frame": [null, null]
+            },
+            {
+                "calculate": "datum.IMDB_Rating - datum.AverageRating",
+                "as": "DifferenceFromMean"
+            },
+            {
+                "filter": "datum.DifferenceFromMean > 2.5"
+            }
+        ],
+        "mark": { "type": "bar", "clip": true },
+        "encoding": {
+            "x": {
+                "field": "Title",
+                "type": "ordinal"
+            },
+            "y": {
+                "field": "IMDB_Rating",
+                "type": "quantitative",
+                "scale": { "domain": [7, 10] },
+                "axis": { "title": "IMDB rating" }
+            }
+        }
+    }]
+}
 ```
 
-### Percent Difference from Total
+### Percent Difference from mean
+
+The example above, could be adjusted to be the percent difference instead of the absolute difference, by adjusting the window transform to be in the format below.
 
 ```json
   {
     "window": [{
-        "op": "sum",
+        "op": "mean",
         "field": "x",
-        "as": "total"
+        "as": "average"
     }],
     "frame": [null, null]
   },
   {
-    "calculate": "(datum.total - datum.x)/dataum.total",
-    "as": "Percent difference from total"
+    "calculate": "(datum.total - datum.x)/dataum.average * 100",
```

Comment:
"dataum"

also the example above doesn't have field "x"

Please make sure to organize sections in a sensible way.